code401challenges/src/main/java/linked_list/LinkedList.java

```diff
@@ -78,6 +78,30 @@ public void insertAfter(int value, int newVal){
 
     }
 
+//  Returns node in k-th value from the end of a linked list.
+    public int insertK(int k){
+        int length = 0;
+//  exception to deal with null pointer values
+        int exception = -1;
```

Comment:
Food for thought: How does the user know if the value kth from the end is `-1` or if this is returning the exception?